stablehlo/transforms/Passes.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2022 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef STABLEHLO_TRANSFORMS_PASSES_H
+#define STABLEHLO_TRANSFORMS_PASSES_H
+
+#include <memory>
+
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir {
+namespace vhlo {
```

Comment:
Do we anticipate any other passes? The reason the file was originally named `CompatibilityPasses` was because all transformations of StableHLO were strictly for compatibility purposes, since we are a shallow opset with no optimizations.

If we think we'll eventually have other legalizations to other dialects (maybe to TCP if it gets moved to upstream eventually?) then I think it is right to call this `stablehlo`, otherwise would tend towards `vhlo` for the above reason.

Reply:
https://github.com/tensorflow/mlir-hlo/blob/0a819de96902c7a84a1fca6fb33abaded3569b6d/mhlo/transforms/rewriters.h#L182

Changed. Note this is a slight deviation from MHLO where the target dialect is the namespace used. We could split this so passes are in `stablehlo` and `populate` methods are in `vhlo`, but I'm not sure that subtlety is worth it.